examples/typescript/components/http-hello-world/http-hello-world.ts

```diff
@@ -14,16 +15,21 @@ function handle(req: IncomingRequest, resp: ResponseOutparam) {
 
   // Access the outgoing response body
   let outgoingBody = outgoingResponse.body();
-  // Create a stream for the response body
-  let outputStream = outgoingBody.write();
-  // // Write hello world to the response stream
-  outputStream.blockingWriteAndFlush(
-    new Uint8Array(new TextEncoder().encode("Hello from Typescript!\n"))
-  );
+  {
+    // Create a stream for the response body
+    let outputStream = outgoingBody.write();
+    // Write hello world to the response stream
+    outputStream.blockingWriteAndFlush(
+      new Uint8Array(new TextEncoder().encode("Hello from Typescript!\n"))
+    );
+    // @ts-ignore: This is required in order to dispose the stream before we return
+    outputStream[Symbol.dispose]();
+  }
 
   // Set the status code for the response
   outgoingResponse.setStatusCode(200);
-
+  // Finish the response body
+  OutgoingBody.finish(outgoingBody, undefined);
```

Comment:
It either has to be undefined, a `new Fields()`, or a `//@ts-lint ignore` directive unfortunately. We might have something in tsconfig.json that we can tweak but I asked @lachieh the other day on this and because the type signature is `trailers: Trailers | undefined` you have to specify both (if it was `trailers?: Trailers` we could omit it)

Reply:
TIL ty!